src/types.tsx

```diff
@@ -0,0 +1,6 @@
+type Attempt = {
+  x: number
+  y: string
```

Comment:
What's up with these property names?

Reply:
I believe when I start doing the charting with Victory that the data object I pass to the Victory components needs to be an object with properties x and y.  I will experiment more when I get there, but all my simple tests required it

Reply:
You could probably rename them at the point of injection into the Victory chart. Not sure if that transition into the API is elegant, but it would allow you to keep your own types a bit clearer while you are construction them and building up the objects.
```
<MyChart data={{x: attempt.number, y: attempt.timestamp}} />
```